### ISSUE TITLE
New plunker editor - where is the version history?

### ISSUE THREAD

**Issue by nirch** [plnkr/feedback#79]
I am looking for the option to view version history and go back to old versions of my plunk in the new plunker editor and can't find it.

It is there? If not, it would be great to have it back since it is an important functionallity in teaching and showing the evolution of a sample.

Thanks!

**Comment by ggoodman** [comment]
@nirch can you tell me a bit more about how you use this feature and your use-case? In redesigning it, it would be great to optimize it for how people actually use it!

Thanks for your feedback.

**Comment by rr20eid10** [comment]
Yes, I agree with @nirch --the version history was a great feature for the use case he explains above -- as well as  naming the plunk so that I know what it is.  Why were these removed from plain view?  Thanks!

**Comment by nirch** [comment]
@ggoodman I am teaching a web development course and I am using plunker to show and share code samples with my class. The version history feature is a great tool for my students to view the evolution of the plunk (of the sample) instead of just being able to see the final result.

Let me know if you need more details.

I love the console feature in the preview of the new version!

**Comment by ggoodman** [comment]
From @dag93tt:

> It would also be great to be able to label or make comment in the title of each previous version so you can track your progress easily!

This seems like a marvellous idea to me.
